internal/types/ordinals.go

```diff
@@ -0,0 +1,35 @@
+package types
+
+import "encoding/json"
+
+type UTXORequest struct {
+    Txid string `json:"txid"`
+    Vout int    `json:"vout"`
+}
+
+type SafeUTXO struct {
+    TxId string `json:"txId"`
+    Brc20 bool  `json:"brc20"`
```

Comment:
```suggestion
    Inscription bool  `json:"brc20"`
```

BRC-20 is just one of the protocols.